<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reporte_kardex</class>
 <widget class="QWidget" name="reporte_kardex">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="central_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="backgroundcolor_frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(29, 53, 87);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="content_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QTableWidget" name="tabla_entradas_kardex">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>60</y>
              <width>711</width>
              <height>261</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QTableWidget {
        background-color: #F1FAEE;
        color: #1D3557;
        border: 2px solid #457B9D;
        border-radius: 5px;
        font-size: 14px;
        gridline-color: #457B9D;
    }
    QTableWidget::item {
        border: 1px solid #457B9D;
    }
    QTableWidget::item:selected {
        background-color: #457B9D;
        color: #F1FAEE;
    }
    QHeaderView::section {
        background-color: #457B9D;
        color: #F1FAEE;
        padding: 5px;
        border: 2px solid #1D3557;
        border-radius: 5px;
        font-size: 14px;
    }</string>
            </property>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Producto</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Cantidad</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Fecha</string>
             </property>
            </column>
           </widget>
           <widget class="QPushButton" name="import_excel_kardex">
            <property name="geometry">
             <rect>
              <x>610</x>
              <y>600</y>
              <width>101</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
        background-color: #457B9D;
        color: #F1FAEE;
        border: 2px solid #1D3557;
        border-radius: 5px;
        padding: 5px;
        font-size: 14px;
    }
    QPushButton:hover {
        background-color: #1D3557;
        border: 2px solid #457B9D;
    }
    QPushButton:pressed {
        background-color: #1D3557;
        border: 2px solid #457B9D;
        padding-left: 7px;  /* Para dar un efecto de &quot;presi√≥n&quot; */
    }</string>
            </property>
            <property name="text">
             <string>excel</string>
            </property>
           </widget>
           <widget class="QComboBox" name="productos_kardex">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>10</y>
              <width>211</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
        background-color: #F1FAEE;
        color: #1D3557;
        border: 2px solid #457B9D;
        border-radius: 5px;
        padding: 5px;
        font-size: 14px;
    }
    QComboBox::drop-down {
        border: none;
    }
    QComboBox::down-arrow {
        image: url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-expand-arrow-24.png);
    }
    QComboBox QAbstractItemView {
        border: 2px solid #457B9D;
        selection-background-color: #457B9D;
    }</string>
            </property>
           </widget>
           <widget class="QTableWidget" name="tabla_salidas_kardex">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>330</y>
              <width>711</width>
              <height>261</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QTableWidget {
        background-color: #F1FAEE;
        color: #1D3557;
        border: 2px solid #457B9D;
        border-radius: 5px;
        font-size: 14px;
        gridline-color: #457B9D;
    }
    QTableWidget::item {
        border: 1px solid #457B9D;
    }
    QTableWidget::item:selected {
        background-color: #457B9D;
        color: #F1FAEE;
    }
    QHeaderView::section {
        background-color: #457B9D;
        color: #F1FAEE;
        padding: 5px;
        border: 2px solid #1D3557;
        border-radius: 5px;
        font-size: 14px;
    }</string>
            </property>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Producto</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Cantidad</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Fecha</string>
             </property>
            </column>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
